lib/v2/taptap/utils.js

```diff
@@ -35,6 +35,7 @@ const textPost = async (appId, topicId, lang = 'zh_CN') => {
 const videoPost = (video) =>
     art(path.join(__dirname, 'templates/videoPost.art'), {
         intro: video?.intro?.text,
+        previewUrl: video?.video_resource.preview_animation.url,
         videoUrl: video?.url,
         posterUrl: video?.raw_cover.url,
```

Comment:
```suggestion
        previewUrl: video?.video_resource.preview_animation.original_url,
```

- `original_url` has a higher resolution
- remove unused property